src/corelibs/U2View/src/ov_sequence/PanViewRows.cpp

```diff
@@ -74,6 +74,15 @@ bool PVRowData::fitToRow(const QVector<U2Region>& location) {
     return true;
 }
 
+bool PVRowData::containsAnnotationWithName(const QString& name) const {
+    for (Annotation* a : annotations) {
```

Comment:
Every row may contain only annotations with the same name. It means that you do not need to iterate all of them (there may be 10k+ annotations in a row like enzymes) and only need to check the first in the list.

Reply:
done